app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,50 @@
+class QuestionsController < ApplicationController
+  before_action :set_question, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @questions = Question.all
+  end
+
+  def show
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def edit
+  end
+
+  def create
+    @question = Question.new(question_params)
+
+    if @question.save
+      redirect_to @question
+    else
+      render :new
+    end
+  end
+
+  def update
+    if @question.update(question_params)
+      redirect_to @question
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    @question.destroy
+    redirect_to questions_path
+  end
+
+  private
+
+  def set_question
+    @question = Question.find(params[:id])
+  end
+
+  def question_params
+    params.require(:question).permit(:title, :body).merge(user: User.last)
```

Comment:
prishlos' dobavit' "merge(user: User.last)", potomu shto toli u menya na noute shto-to rabotaet ne tak, libo eto novaya fishka v rails 5. Obichno esli "Answer" i "User" svyazani, to takoi kastil' ne nugen, no u menya hot' svyazi est', vse ravno pri sozdanii objecta "user_id: nil", poetomu ego yavno ukazivau. Potom kogda 'devise' dobavlu, "User.last" smenu na "current_user.id"